docs/README.md

```diff
@@ -0,0 +1,161 @@
+## Run a full diracx demo
+
+This will allow you to run a demo setup.
+
+The code changes will be reflected in the demo.
+
+Requirement: docker, internet
+
+```bash
+# Clone the diracx repository
```

Comment:
Clone your fork ?

Reply:
What if I just want to try the demo? (and not forking the whole project)